waspc/data/Generator/templates/sdk/client/auth/index.ts

```diff
@@ -0,0 +1,20 @@
+{{={= =}=}}
+export * from './ui'
+{=# isEmailAuthEnabled =}
+export * from './email'
+{=/ isEmailAuthEnabled =}
+{=# isUsernameAndPasswordAuthEnabled =}
+export * from './username'
```

Comment:
Both email and username can contain login and/or signup. So we would be exporting same symbols here. But that is not an issue because two are never enabled at the same time, is that it? Do we have a GH issue to handle this in the future, should we create one?

Reply:
>  But that is not an issue because two are never enabled at the same time, is that it?
 
That's correct 👍 

> Do we have a GH issue to handle this in the future, should we create one?

How do you think we should handle it? Drop username or something like that or unify the two login methods?

Reply:
Not sure -> I don't think anymore we will be dropping the username&pass soon, so I would like into ways to have both.
Either we unify the functions, or we have multiple functions but either export them from different modules `/auth/email`, `auth/username` or modify names `loginWithUsername` `loginWithEmail` or something like that.

Pls create a GH issue, mark it with `auth` and `restructuring` labels and we can further discuss there.

Reply:
I'm not a big fan of the long names.

Why do you think it's problematic if `login` keeps its name but changes its type signature depending on the auth method users have configured?

Reply:
This convo is related to https://github.com/wasp-lang/wasp/issues/1697 so I linked it there also.